digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectDeltaCalculator.scala

```diff
@@ -68,13 +71,59 @@ object ProjectDeltaCalculator {
     ).toSeq
   }
 
+  def orderProjectLinksTopologyByGeometry(list:Seq[ProjectLink]): Seq[ProjectLink] = {
+
+    def linkTopologyByGeomEndPoints(sortedList: Seq[ProjectLink], unprocessed: Seq[ProjectLink]) : Seq[ProjectLink] = {
```

Comment:
I think there should be error state here. If i am not mistaken (and i might be) we force list to be sorted at the end.  It is possible that user has selection that we cannot sort, especially after we add feature where user can add links by pressing shift.